SConstruct

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
```

Comment:
You could try changing the shebang to `#!/usr/bin/env python3`.

Reply:
This shebang is mostly to help IDEs with auto-completion, we don't actually run SConstruct via Python, it's parsed by SCons itself.

Reply:
True. It's the other way around. Changing the shebang at the beginning of the scons script itself to `#! /usr/bin/python3` makes it work (and there is no need to install the latest scons using pip3!)

Reply:
If you use `/usr/bin/python3`, this means the user-chosen Python version (e.g. with [pyenv](https://github.com/pyenv/pyenv)) will be ignored :slightly_frowning_face:

Reply:
@Calinou This change is only needed for distributions that still have Python v2 as the default i.e. python points to python2 and for versions of scons that are have the shebang pointing to /usr/bin/python2 as for example my scons version 3.1.1 on Ubuntu was doing.

I'll submit an update to the documentation to reflect this.